.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,54 @@
-**What this PR does / why we need it**:
-
-**Which issue(s) this PR fixes**:
 <!--
-Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+If this pull request is just an idea or POC, or is not ready for review, select "Create draft pull request" (https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests)
+instead of "Create pull request"
 -->
-Fixes #
 
-**Describe testing done for PR**:
+### What this PR does / why we need it
+
+### Which issue(s) this PR fixes
 <!--
-Example: Created vSphere workload cluster to verify change.
+     Usage: Fixes #<issue number>.
+
+     Unless the PR is for a trivial change (e.g. fixing a typo), consider opening an issue first
+     (and reference it here) so that the problem the PR addresses can be discussed independently of
+     the solutions proposed by this PR.
 -->
 
-**Special notes for your reviewer**:
+Fixes #
+
+### Describe testing done for PR
+
+<!-- Example: Created vSphere workload cluster to verify change. -->
 
-**Release note**:
+### Release note
 <!--
-See https://github.com/vmware-tanzu/tanzu-framework/blob/main/docs/release/release-notes.md#does-my-pull-request-need-a-release-note
-for more details.
+     Please add a short text (limit to 1 to 2 sentences if possible) in the release-note block below if
+     there is anything in this PR that is worthy of mention in the next release.
 
-Please add a short text in the release-note block below (or "NONE" if not applicable)
-if there is anything in this PR that is worthy of mention in the next release.
+     See https://github.com/vmware-tanzu/tanzu-framework/blob/main/docs/release/release-notes.md#does-my-pull-request-need-a-release-note
```

Comment:
The referenced docs mention adding the appropriate kind/label. Is that how release notes get categorized? Should those docs also be updated to state that the label will be added as part of the triage process instead?